OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -41,6 +41,10 @@ public class RallyPointInfo : ITraitInfo
 
 		public readonly CVec Offset = new CVec(1, 3);
 
+		[NotificationReference("Speech")]
+		[Desc("The speech notification to play when setting a new rallypoint.")]
+		public readonly string EstablishedNotification = null;
```

Comment:
Can we please call this just `Notification`, to match other traits? The "established" part doesn't add much meaning outside of the specific RA2 voice string.

Reply:
As there probably won't be another one I changed it.